lib/tools/te_enum.h

```diff
@@ -296,6 +296,40 @@ extern int te_enum_translate(const te_enum_trn trn[],
                              int value, te_bool reverse,
                              int unknown_val);
 
+/**
+ * A mapping between a one one-bit mask from one bitmask and its
+ * corresponding one-bit mask in another bitmask.
+ *
+ * An array of mappings should end with TE_ENUM_BITMASK_CONV_END.
+ */
+typedef struct te_enum_bitmask_conv {
+    /** Left-hand side one-bit mask. */
+    uint64_t one_bit_mask_from;
+    /** Right-hand side one-bit mask. */
+    uint64_t one_bit_mask_to;
```

Comment:
This one-bit requirement does not actually affect translation nor is it enforced. I'd say that the requirement shall instead be that the bitmasks shall not be overlapping.